audio-worklet/src/bitcrusher-demo.js

```diff
@@ -141,18 +167,20 @@ class BitcrusherDemo {
    */
   setBitDepth(value) {
     if (value < 1) throw 'The minimum bit depth rate is 1.';
-    this.paramBitDepth_.value = value;
+    if (this.useWorkletAndScriptProcessor_)
+      this.paramBitDepth_.value = parseFloat(value);
```

Comment:
Why do you need a `parseFloat` here but not for setting the script processor bit depth?

Similarly below for setting the reduction value.